lib/std/special/test_runner.zig

```diff
@@ -35,6 +35,8 @@ pub fn main() void {
         .dont_print_on_dumb = true,
     };
     const root_node = progress.start("Test", test_fn_list.len);
+    root_node.completeOne();
```

Comment:
incorrect API usage; it didn't actually complete one task, did it?

Reply:
It did not. But the test runner is executing the first test with `[0/x]`, where x is the number of tests. I guess the first test should be displayed as `[1/x]`.

Is there another way to accomplish that?

Reply:
Would you prefer a `start_with_index_one` flag in the progress struct? This would allow to choose between 'start with 0' or 'start with 1'.

Reply:
I don't think such thing should be configurable. I'm OK with changing the display to be more intuitive however.

Reply:
Ok, cool. I'll change that.